connectivity/check/metrics_test.go

```diff
@@ -0,0 +1,125 @@
+package check
+
+import (
+	"reflect"
+	"testing"
+
+	dto "github.com/prometheus/client_model/go"
+	prommodel "github.com/prometheus/client_model/go"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParsePromMetrics(t *testing.T) {
+	input := `
+# HELP isovalent_external_dns_proxy_policy_l7_total Number of total proxy requests handled
+# TYPE isovalent_external_dns_proxy_policy_l7_total counter
+isovalent_external_dns_proxy_policy_l7_total{rule="forwarded"} 132
+isovalent_external_dns_proxy_policy_l7_total{rule="received"} 132
+`
```

Comment:
Could we please use metrics here that are not proprietary, i.e. that are available in an  Cilium OSS version?

Reply:
Oups! Yes of course!

Reply:
Fixed.